src/authorization-response/AuthorizationResponse.ts

```diff
@@ -120,7 +120,9 @@ export class AuthorizationResponse {
         presentationDefinitions,
         presentations: wrappedPresentations,
         verificationCallback: verifyOpts.verification.presentationVerificationCallback,
-        opts: { ...responseOpts.presentationExchange, hasher: verifyOpts.hasher },
+        opts: { ...responseOpts.presentationExchange,
+          presentationSubmission: (responseOpts.presentationExchange.presentationSubmission ?? authorizationResponsePayload.presentation_submission),
```

Comment:
assertValidVerifiablePresentations has this check
  ```
  } else if (args.presentationDefinitions && !args.opts.presentationSubmission) {
    throw new Error(`No presentation submission present. Please use presentationSubmission opt argument!`);
  }
  ```
  So we need opts.presentationSubmission